edgelet/iotedge/src/main.rs

```
@@ -323,6 +355,76 @@ fn run() -> Result<(), Error> {
             tokio_runtime.block_on(Logs::new(id, options, runtime()?).execute())
         }
         ("version", _) => tokio_runtime.block_on(Version::new().execute()),
+        ("cssbundle", Some(args)) => {
+            let location = args.value_of("location").unwrap_or_default();
+            let tail = args
+                .value_of("tail")
+                .and_then(|a| a.parse::<LogTail>().ok())
+                .unwrap_or_default();
+            let since = args
+                .value_of("since")
+                .and_then(|s| parse_since(s).try_into().ok())
+                .unwrap_or_default();
+            let options = LogOptions::new()
+                .with_follow(false)
+                .with_tail(tail)
+                .with_since(since);
+            tokio_runtime.block_on(Bundle::new(options, location, runtime()?).execute())
+        }
         (command, _) => tokio_runtime.block_on(Unknown::new(command.to_string()).execute()),
     }
 }
+
+fn parse_since(since: &str) -> i64 {
+    if let Ok(datetime) = DateTime::parse_from_rfc3339(since) {
+        datetime.timestamp()
+    } else if let Ok(epoch) = since.parse() {
+        epoch
+    } else if let Ok(duration) = parse_duration::parse(since) {
+        (Local::now() - Duration::nanoseconds(duration.as_nanos().try_into().unwrap_or_default()))
+            .timestamp()
+    } else {
+        /* Default to 1 day ago */
+        println!("Flag since ({}) not recognized. Defaulting to 1 day", since);
```

This should return a fatal error instead of defaulting.